src/main/java/seedu/address/storage/CsvManager.java

```diff
@@ -0,0 +1,70 @@
+package seedu.address.storage;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+import com.opencsv.CSVWriter;
+
+import javafx.collections.ObservableList;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.model.property.Buyer;
+import seedu.address.model.property.Property;
+
+/**
+ * Manages exporting to or importing from csv.
+ */
+public class CsvManager {
```

Comment:
should everything here be static?

Reply:
mmm CsvManager is now in Storage, and storage does not seem to use static.